app/main/views/templates.py

```diff
@@ -628,12 +628,9 @@ def delete_service_template(service_id, template_id):
         else:
             raise e
 
+    flash(["Are you sure you want to delete ’{}’?".format(template['name']), message], 'delete')
```

Comment:
```suggestion
    flash(["Are you sure you want to delete ‘{}’?".format(template['name']), message], 'delete')
```
Thanks for trying to use the right one 😉

Reply:
Uhh, I will fix it, and meanwhile here's a badge for me :P

![Uploading you_tried_button-rf97e1ee4c6484f658c9d8d4561f65898_k94rf_307.jpg…]()

Reply:
fixed 😊